app-emulation/fuse/fuse-1.6.0.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Free Unix Spectrum Emulator by Philip Kendall"
+HOMEPAGE="http://fuse-emulator.sourceforge.net"
+SRC_URI="mirror://sourceforge/fuse-emulator/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="alsa ao backend-fbcon backend-sdl backend-svga backend-X gpm joystick memlimit png pulseaudio xml"
+
+# Only one UI back-end can be enabled at a time
+# At most one audio driver can be enabled at a time
+REQUIRED_USE="?? ( backend-fbcon backend-sdl backend-svga backend-X )
+	?? ( alsa ao pulseaudio )"
```

Comment:
This seems like an awful restriction, as at least people with `pulseaudio` will most likely have `alsa` enabled too. I'd suggest removing the `REQUIRED_USE` for this, and just prioritize the engine during `src_configure`. If `+pulseaudio` then always choose that, if `-pulseaudio +alsa` then choose alsa, and `-pulseaudio -alsa +ao` choose ao.